incubator/fluentd-elasticsearch/templates/daemonset.yaml

```diff
@@ -68,6 +68,40 @@ spec:
 {{- if .Values.extraVolumeMounts }}          
 {{ toYaml .Values.extraVolumeMounts | indent 8 }}
 {{- end }}
+{{- if .Values.livenessProbe.enabled }}       
```

Comment:
Why would you even want to make it optional?

Reply:
Because I'm not sure all the users want to have this probe, what I can do is change the default value to true. WDYT?